### PR TITLE
Wallet List Modal Filter Tokens

### PR DESCRIPTION
**Pull request by amanalili**
Task: Wallet List - Tokens to appear as standalone wallets (https://app.asana.com/0/361770107085503/1155557910296414)

Added token filters and hiding of ethereum parent wallet on the Wallet Selector Modal

#### PR Requirements

If you have made **any** visual changes to the GUI. Make sure you have:
- [x] Tested on iOS Tablet
- [x] Tested on small Android

### CODE REVIEW

File: src/components/common/CryptoExchangeWalletListRow.js [hunk 1]
```diff
@@ -110,12 +112,21 @@ class CryptoExchangeWalletListRow extends Component<Props, LocalState> {
...
-            if (property !== this.props.excludedCurrencyCode && !this.props.excludedTokens.includes(property)) {
+
+            // Token Filters
+            const nameString = name.toLowerCase()
+            const currencyNameString = token ? token.currencyName.toLowerCase() : ''
+            const currencyCodeString = token ? token.currencyCode.toLowerCase() : ''
+            if (
+              property !== this.props.excludedCurrencyCode &&
+              !this.props.excludedTokens.includes(property) &&
+              (nameString.includes(inputLowerCase) || currencyNameString.includes(inputLowerCase) || currencyCodeString.includes(inputLowerCase))
```
Comment (paullinator) [inline src/components/common/CryptoExchangeWalletListRow.js]: needs also `inputLowerCase === '' ||`

File: src/components/common/CryptoExchangeWalletListRow.js [hunk 2]
```diff
@@ -26,7 +26,8 @@ export type OwnProps = {
   excludedCurrencyCode: Array<string>,
   onTokenPress({ id: string, currencyCode: string }): void,
   excludedTokens: Array<string>,
-  disableZeroBalance: boolean
+  disableZeroBalance: boolean,
+  input: string
```
Comment (paullinator) [inline src/components/common/CryptoExchangeWalletListRow.js]: Change `input` to `searchFilter`

File: src/components/common/CryptoExchangeWalletListRow.js (hunk 1 above)
Comment (amanalili) [reply src/components/common/CryptoExchangeWalletListRow.js]: fixed

File: src/components/common/CryptoExchangeWalletListRow.js (hunk 2 above)
Comment (amanalili) [reply src/components/common/CryptoExchangeWalletListRow.js]: fixed

**Pull request merged**